src/components/metadata/form/index.js

```diff
@@ -440,11 +446,19 @@ const MetadataForm = ({ loading, showErrorAnnouncer, ...props }) => {
 
     const baseId = ids.EDIT_METADATA_FORM;
 
+    if (loadingError) {
+        return (
+            <WrappedErrorHandler baseId={baseId} errorObject={loadingError} />
+        );
+    }
+
+    const { avus, "irods-avus": irodsAVUs } = metadata;
+
     return (
         <>
             <Formik
                 enableReinitialize
-                initialValues={{ ...metadata }}
+                initialValues={{ avus, "irods-avus": irodsAVUs }}
```

Comment:
I guess the DE's GXT code was sanitizing the service response before passing it to the old React code, since the metadata endpoint also returns a `path` in its response that is not acceptable in the metadata add/update endpoints. So this form is now explicitly only using the keys required in the form and submission.